src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -49,18 +51,18 @@
     private final BigQueryPdao bigQueryPdao;
     private final RestHighLevelClient client;
 
-    private final Map<String, String> columnReplacements = Map.of(
-        "biosample_id", "dct:identifier",
-        "donor_id", "prov:wasDerivedFrom",
-        "disease", "TerraCore:hasDisease",
-        "genus_species", "TerraCore:hasOrganismType",
-        "organ", "TerraCore:hasAnatomicalSite",
-        "library_construction_method_text", "TerraCore:hasLibraryPrep",
-        "sex", "TerraCore:hasSex",
-        "project_title", "dct:title",
-        "cell_type", "TerraCore:hasSelectedCellType",
-        "organism_age_unit", "TerraCore:hasAgeUnit"
-    );
+    private final BiMap<String, String> columnReplacements = new ImmutableBiMap.Builder<String, String>()
```

Comment:
If you need both a `Map` and a set of its values, you can create a `Map` and then use `new HashSet<>(maps.values())` to make a set of its values. I don't think the general features of `BiMap` are needed here. The both the map and the values are static data so I would make them static finals too.

Reply:
Oh yeah, that's true. I've switched it to use the `Map` and `map.values()`